collect/cache/cuckooSentCache.go

```diff
@@ -28,18 +28,29 @@ type keptTraceCacheEntry struct {
 	reason         uint32 // which rule was used to decide to keep the trace
 }
 
-func NewKeptTraceCacheEntry(trace *types.Trace) *keptTraceCacheEntry {
-	if trace == nil {
+type TraceInfo interface {
```

Comment:
This is the interface for both data sent to the central repository and to honeycomb